<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="1" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="33" started-at="2018-07-21T20:51:42Z" finished-at="2018-07-21T20:51:42Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="33" started-at="2018-07-21T20:51:42Z" finished-at="2018-07-21T20:51:42Z">
      <class name="test.TestNG_TestResult">
        <test-method status="FAIL" signature="method1()[pri:0, instance:test.TestNG_TestResult@77ec78b9]" name="method1" duration-ms="9" started-at="2018-07-21T20:51:42Z" finished-at="2018-07-21T20:51:42Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at test.TestNG_TestResult.method1(TestNG_TestResult.java:12)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:test.TestNG_TestResult@77ec78b9]" name="afterMethod" is-config="true" duration-ms="1" started-at="2018-07-21T20:51:42Z" finished-at="2018-07-21T20:51:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=method1 status=FAILURE method=TestNG_TestResult.method1()[pri:0, instance:test.TestNG_TestResult@77ec78b9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="method2()[pri:0, instance:test.TestNG_TestResult@77ec78b9]" name="method2" duration-ms="2" started-at="2018-07-21T20:51:42Z" finished-at="2018-07-21T20:51:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- method2 -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:test.TestNG_TestResult@77ec78b9]" name="afterMethod" is-config="true" duration-ms="0" started-at="2018-07-21T20:51:42Z" finished-at="2018-07-21T20:51:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=method2 status=SUCCESS method=TestNG_TestResult.method2()[pri:0, instance:test.TestNG_TestResult@77ec78b9] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- test.TestNG_TestResult -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
